packages/engine/Source/Scene/Model/CustomShader.js

```diff
@@ -269,6 +280,34 @@ function buildUniformMap(customShader) {
   return uniformMap;
 }
 
+function buildUniformExtraInfoMap(customShader) {
+  const uniforms = customShader.uniforms;
+  const uniformExtraInfoMap = {};
+  for (const uniformName in uniforms) {
+    if (uniforms.hasOwnProperty(uniformName)) {
```

Comment:
[`hasOwn()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) should be preferred over `hasOwnProperty()`

Reply:
I see! While I think it makes sense to use `hasOwn()` the whole codebase is using `hasOwnProperty()` in multiple places.
There's only one occurrence of `hasOwn()` and that's in the gulpfile and not the "real" code.

I guess this is just because Cesium did not adopt `hasOwn()` yet.
Should we use `hasOwnProperty()` to stay in line with the rest of the code base or shoudl we use `hasOwn()` for our changes? I'm not sure about this.

Reply:
Hmm since we don't plan to create this PR against the original repo, I think it would be fine to "break" with the code style in this case and use `hasOwn`. But I would also be fine with keeping `hasOwnProperty` 😄
Maybe @anne-gropler has an opinion on this?

Reply:
> Do we have inheritance in the uniforms

I don't know! 😅 
However, I think the same is true for `hasOwnProperty()`:
> returns a boolean indicating whether this object has the specified property as its own property (as opposed to inheriting it)
(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty)

So I guess this doesn't make a difference? 🙈

Reply:
ooh okay, I thought that was a difference between hasOwn and hasOwnProperty :D 

Yes, then I think we should use `hasOwn`, at least for our code here.

Reply:
I just changed it to `hasOwn()`!
I think it's fine here! :D